dev-qt/qt3d/qt3d-5.15.2-r10.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KDE_ORG_COMMIT=7edec6e014de27b9dd03f63875c471aac606a918
+inherit qt5-build
+
+DESCRIPTION="3D rendering module for the Qt5 framework"
+
+if [[ ${QT5_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64 ~arm64 ~x86"
+fi
+
+# TODO: tools
+IUSE="gamepad gles2-only qml vulkan"
+
+RDEPEND="
+	~dev-qt/qtconcurrent-${PV}
+	~dev-qt/qtcore-${PV}
+	~dev-qt/qtgui-${PV}:5=[vulkan=]
+	~dev-qt/qtnetwork-${PV}
+	>=media-libs/assimp-4.0.0
+	gamepad? ( ~dev-qt/qtgamepad-${PV} )
+	qml? ( ~dev-qt/qtdeclarative-${PV}[gles2-only=] )
+"
+DEPEND="${RDEPEND}
+	vulkan? ( dev-util/vulkan-headers )
```

Comment:
Just checking: we include it in RDEPEND for qtgui, qtwayland presumably so that consumers of those don't die on the missing header when they hit the ```#include```. No need here?